src/components/label/style.css

```diff
@@ -0,0 +1,14 @@
+:host>div {
```

Comment:
with the removal of the div remove the div here from the selector

Reply:
\+